keyvalue/client/pom.xml

```diff
@@ -35,7 +35,7 @@
     <!-- Internal dependencies -->
     <dependency>
       <groupId>org.alluxio</groupId>
-      <artifactId>alluxio-core-client</artifactId>
+      <artifactId>alluxio-core-client-hdfs</artifactId>
```

Comment:
I think we also use native client. in this case, we should also reference `alluxio-core-client-native`